PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterCharacterScripts/Sound.rbxmx

```diff
@@ -83,7 +85,15 @@ local Sounds = {}
 do
 	local Figure = script.Parent.Parent
 	Head = Figure:WaitForChild("Head")
-	Humanoid = Figure:WaitForChild("Humanoid")
```

Comment:
I understand we want a type of humanoid and thats why you changed this, but why just for humanoid? Head doesn't wait for an actual head